core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SampleTissuePiece.java

```diff
@@ -0,0 +1,17 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+import uk.ac.bbsrc.tgac.miso.core.data.type.TissuePieceType;
+
+public interface SampleTissuePiece extends SampleTissueProcessing {
+
+  public static final String SUBCATEGORY_NAME = "Tissue Piece";
+
+  TissuePieceType getTissuePieceType();
+
+  void setTissuePieceType(TissuePieceType type);
+
+  Integer getSlidesConsumed();
```

Comment:
All types of Tissue Piece have 'slides consumed'?

Reply:
Yes. That's ultimately what they really want out of this change. Currently curls do not track the number of slides consumed, and they should.